docs/integrations/sources/recurly.md

```diff
@@ -0,0 +1,37 @@
+# Recurly
+
+## Overview
+
+The Recurly source supports Full Refresh syncs. That is, every time a sync is run, Airbyte will copy all rows in the tables and columns you set up for replication into the destination in a new table.
+
+### Output schema
+
+Several output streams are available from this source (accounts, plans, invoices, transactions, etc.).
```

Comment:
Could you: 

1. List the entities output (instead of "etc.")
2. Add a URL to the recurly docs that helps the user understand what each of these entities mean

```suggestion
Several output streams are available from this source: 
* accounts
* ....

```